service/collector/set.go

```diff
@@ -1,19 +1,21 @@
 package collector
 
 import (
-	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
 	"github.com/giantswarm/exporterkit/collector"
+	"github.com/giantswarm/k8sclient"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
-	"k8s.io/client-go/kubernetes"
 
 	clientaws "github.com/giantswarm/aws-operator/client/aws"
 )
 
+const (
+	collidingOperatorLabel = "app.kubernetes.io/version"
```

Comment:
`collidingOperatorLabel` is a weird name IMO. We also have `labels` package.

Also more importantly CRs won't have this label. Only resources created with aws-operator chart will have it (Deployment etc.). CRs have labels like `aws-operator.giantswarm.io/version`. This is because the same object can be reconciled by multiple operators.

Reply:
Noted. Would `awsOperatorVersionLabel` work for you?

Reply:
I think what Pawel meant was that we do not need this thing at all. If you need a label it is in the labels package. See https://github.com/giantswarm/aws-operator/blob/master/pkg/label.

Reply:
Oooh, fancy. Sure.